### DISCUSSION TITLE
Help for Vercel deployment

### DISCUSSION THREAD

**Discussion by Synomega** [colinwilson/lotusdocs#108] [Q&A]
Hi,

I'd like to deploy to vercel but I don't know how, I tried with the ExempleSite and with https://github.com/colinwilson/lotusdocs.dev

But I got errors on vercel.

Thanks in advance.

**Comment by colinwilson** [comment]
Apologies. I'm currently behind on the documentation but I'll put this up on [lotusdocs.dev](https://lotusdocs.dev/deployment) before the weekend. Will update here once it's live

**Reply by Synomega** [reply]
Thanks for your information if it can help you, when I try to deploy on Vercel here the error :

`
INFO 2023/10/04 14:10:25 syncing static files to /home/runner/work/lotusdocs/lotusdocs/exampleSite/public/
Total in 9 ms
Error: Error building site: "/home/runner/work/lotusdocs/lotusdocs/exampleSite/content/de/docs/quickstart.md:[7](https://github.com/myrepo/lotusdocs/actions/runs/6407209201/job/17393508220#step:5:8)5:1": failed to extract shortcode: template for shortcode "alert" not found
Error: Process completed with exit code 255.
`
